### PR TITLE
Improve nearby guard interactions

### PR DESCRIPTION
**Pull request by someaddons**
# Changes proposed in this pull request:
Improve nearby guard interactions

Review please

### CODE REVIEW

File: src/api/java/com/minecolonies/api/util/constant/TranslationConstants.java [hunk 1]
```diff
@@ -412,21 +412,25 @@
     @NonNls
     public static final String OUT_OF_COLONY                                                        = "com.minecolonies.coremod.workorder.outofcolony";
     @NonNls
-    public static final String NO                                                                   = "com.minecolonies.coremod.entity.citizen.no.";
+    public static final String NO                                              = "com.minecolonies.coremod.entity.citizen.no.";
```
Comment (ravenbuilder934) [inline src/api/java/com/minecolonies/api/util/constant/TranslationConstants.java:415]: ..Why did you change the whitespace?

Comment (someaddons) [reply src/api/java/com/minecolonies/api/util/constant/TranslationConstants.java:415]: thats autoformatting of our defined formatting style xD

Comment (ravenbuilder934) [reply src/api/java/com/minecolonies/api/util/constant/TranslationConstants.java:415]: but it's not changing the rest of the file?

File: src/main/java/com/minecolonies/coremod/colony/managers/BuildingManager.java [hunk 2]
```diff
@@ -610,18 +612,10 @@ public boolean hasGuardBuildingNear(final IBuilding building)
...
-            return false;
-        }
-
-        for (final BlockPos buildingPos : closeCap.getAllClaimingBuildings().get(colony.getID()))
+        for (final IBuilding colonyBuilding : getBuildings().values())
         {
-            final IBuilding guardbuilding = colony.getBuildingManager().getBuilding(buildingPos);
-            if (guardbuilding instanceof AbstractBuildingGuards || guardbuilding instanceof BuildingBarracks)
+            if (colonyBuilding instanceof IGuardBuilding && (colonyBuilding.getClaimRadius(colonyBuilding.getBuildingLevel()) * 16) > building.getID()
+                                                                                                                                        .manhattanDistance(colonyBuilding.getID()))
```
Comment (Raycoms) [inline src/main/java/com/minecolonies/coremod/colony/managers/BuildingManager.java]: I'd put the && on a seperate line, else it's kind unreadable

**Review by Raycoms — DISMISSED** [review]
